plugins/scaffolder/report-alpha.api.md

```diff
@@ -379,5 +414,20 @@ export type TemplateWizardPageProps = {
   };
 };
 
+// @alpha (undocumented)
+export const useFormDecorators: ({
```

Comment:
hmm, should this one really be exported?

Reply:
Hmm maybe not at the moment, you can provide your own `TemplateWizardPage` and this would need to be a building block of it. Will remove it for now.